src/window/window_wrapper.rs

```diff
@@ -137,7 +146,7 @@ impl WinitWindowWrapper {
             font_changed_last_frame: false,
             saved_inner_size,
             saved_grid_size: None,
-            ime_enabled,
+            ime_enabled: input_ime,
```

Comment:
This can't be moved without creating a proper new function for the window wrapper, since it does some additional initialization.

But I think that since we now have a proper settings notification system, we wouldn't need to store the `ime_enabled` state in the `WindowWinitWrapper` at all.